example/lib/main.dart

```diff
@@ -189,83 +186,49 @@ class _MyAppState extends State<MyApp> {
     _subscriptionsToDispose.add(_channelStateChangeSubscription);
   }
 
-  // https://github.com/dart-lang/sdk/issues/37498
-  // ignore: missing_return
-  static String opStateDescription(
```

Comment:
This method and the one below presumably stopped being used in a previous bit of work but you're cleaning up now? Just to be clear I understand why these are disappearing - and, presumably, that ideally could have been done under a discrete PR?

Reply:
Yes, exactly correct.

It was removed in https://github.com/ably/ably-flutter/pull/202 and was some code that I didn't remove.